packages/babel-plugin-proposal-class-properties/test/fixtures/private-loose/static-undefined/exec.js

```diff
@@ -0,0 +1,15 @@
+class Foo {
+  static #bar;
+
+  static test() {
+    return Foo.#bar;
+  }
+
+  test() {
+    return Foo.#bar;
+  }
+}
+
+expect("bar" in Foo).toBe(false)
+expect(Foo.test()).toBe(undefined)
+expect(Foo.test()).toBe(undefined)
```

Comment:
Some of these duplicated tests are weird to me `expect(Foo.test()).toBe(undefined)` - I know they were just copied over but is it supposed to be testing the static and instance methods or just remove it?

Reply:
Maybe it should be `expect(() => Foo.test()).not.toThrow()`?

Reply:
@hzoo, you're right, I'll go ahead and do a second review pass on those tests (was only checking for the output I was seeing). I'm also trying to run test-262 right now against it (having `npm link` issues) and I might add some relevant tests then